src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -19,6 +19,34 @@
 #include <Kokkos_Sort.hpp> // min_max_functor
 #include <Kokkos_View.hpp>
 
+#if defined(KOKKOS_ENABLE_CUDA)
+#if (KOKKOS_COMPILER_CLANG < 900)
+// Clang of version less than 9.0 cannot compile Thrust, failing with errors
+// like this:
+//    <snip>/thrust/system/cuda/detail/core/agent_launcher.h:557:11:
+//    error: use of undeclared identifier 'va_printf'
+// Defining _CubLog here allows us to avoid that code path, however disabling
+// some debugging diagnostics
+//
+// If _CubLog is already defined, we save it into ARBORX_CubLog_save, and
+// restore it at the end
+#ifdef _CubLog
+#define ARBORX_CubLog_save _CubLog
```

Comment:
I really dislike that we don't use indentation for preprocessor directives, i.e. `IndentPPDirectives`.

Reply:
I tried that, but it was really bad. Partly because we have the initial `#ifndef HEADER` thing, which makes *everything* nested.

Reply:
I am happy to try in a separate pull request and see what everyone's opinion is.